super_csv/csv_processor.py

```diff
@@ -154,14 +154,14 @@ def get_iterator(self, rows=None, columns=None, error_data=False):
         """
         Export the CSV as an iterator.
         """
+        columns = columns or self.columns
```

Comment:
nit: i think what we're doing would be clearer if we wrote this as
```
if columns is None:
   columns = self.columns
```

Reply:
as soon as I hit enter i started doubting myself, so this is like mega-low nit

Reply:
+1 one to Jansen. The `or` statement is truthy, so anything expressed as false will set columns to `self.colums`. The `is` comparison gives more clarity around the intended case where `self.columns` is to be used.

Reply:
I had written it this way because that's the existing logic (just moved to a new line) but I'm happy to modify it for readability.